src/libcrun/kontain.c

```diff
@@ -57,9 +57,10 @@ int libcrun_kontain_argv(char ***argv, const char **execpath)
       // the command does not exist?  Let the caller handle that.
       return errno;
    }
-   if (strcmp (cmd, KM_BIN_PATH) == 0)
+   if (strcmp (cmd, KM_BIN_PATH) == 0 || strcmp(cmd, INIT_PATH) == 0)
```

Comment:
I'm sort of unhappy about either of these two, as it ultimately allows to run random code masqueraded as either of them. I was hopping there is a definitive way to know that --init was used, and allow it only then.

We can allow this for now to enable investigation of how things should be, temporarily.

Reply:
I will need to check the contents of config.json to see if there is a mount entry for /sbin/docker-init.  I think there will be.
If that is present, then we can do what you want for docker-init.
We should also only allow docker-init as the initial process in the container.  
We don't want to allow them to run "docker exec containerid /sbin/docker-init".